bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/ComponentSensor.java

```diff
@@ -39,18 +43,18 @@
         protected @Nullable String unit_of_measurement;
         protected @Nullable String device_class;
         protected boolean force_update = false;
-        protected int expire_after = 0;
+        protected @Nullable Integer expire_after;
 
         protected String state_topic = "";
+
+        protected @Nullable String json_attributes_topic;
+        protected @Nullable String json_attributes_template;
+        protected @Nullable List<String> json_attributes;
     }
 
     public ComponentSensor(CFactory.ComponentConfiguration componentConfiguration) {
         super(componentConfiguration, ChannelConfiguration.class);
 
-        if (channelConfiguration.force_update) {
-            throw new UnsupportedOperationException("Component:Sensor does not support forced updates");
-        }
```

Comment:
This is supported now?

Reply:
The question is: Does it make any difference?

> force_update
Sends update events even if the value hasn’t changed. Useful if you want to have meaningful value graphs in history.

Do we want the device to tell us such a thing? Would it make any difference to OH,if we would create fake events which repeat the old sensor value again and again?

And even if: Do we want to reject such a device completely?

Reply:
I don't seem to be alone in thinking, that we need to remove that test -> see Issue #7063

Reply:
I also agree @jochen314 and @cpmeister! I don't understand why we should throw any exception if the device supports the force_update functionallity. Go ahead and remove it!